modules/catalog/src/main/java/org/apache/ignite/internal/catalog/storage/UpdateLogImpl.java

```diff
@@ -198,6 +251,10 @@ static ByteArray update(int version) {
         static ByteArray updatePrefix() {
             return ByteArray.fromString("catalog.update.");
         }
+
+        public static ByteArray snapshotVersion() {
+            return ByteArray.fromString("catalog.snapshot.version");
```

Comment:
seems it need to be analogy formed like : DistributionZonesUtil#zonesRecoverableStateRevision ?

Reply:
i meant to move this string into public final String constant